frontend/scripts/react-components/tool/column-selector/column-selector.component.jsx

```diff
@@ -11,13 +11,11 @@ function ColumnSelector({
   handleColumnSelected
 }) {
   return (
-    <div
-      className={cx('js-dropdown c-dropdown -column-selector', {
-        '-hide-only-child': hasSingleElement
-      })}
-    >
+    <div className="js-dropdown c-dropdown">
```

Comment:
I think both these classes aren't necessary. And if they are it's a smell

Reply:
c-dropdown class is used for the layout right now. It will be fixed in the layout PR